services/ymax-planner/README.md

```diff
@@ -58,10 +58,11 @@ yarn start:deps
 ### Local Development
 
 ```bash
-yarn dev
+yarn dev --dry-run
 ```
 
-This starts the development server with hot reloading.
+This starts the development server with hot reloading (omit `--dry-run` to make
+the planner actually send bridge actions rather than just logging them).
```

Comment:
"bridge action" terminology comes from the context of the walletFactory contract. Here it's sort of awkward. Maybe "sign and submit bridge actions" would clarify.

Reply:
Done.